sematic/ui/packages/common/src/pages/RunDetails/FunctionSection.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import Box from '@mui/material/Box';
+import { DateTimeLong, Duration } from 'src/component/DateTime';
+import Docstring from 'src/component/Docstring';
+import Headline from 'src/component/Headline';
+import ImportPath from 'src/component/ImportPath';
+import MoreVertButton from 'src/component/MoreVertButton';
+import PipelineTitle from 'src/component/PipelineTitle';
+import RunReferenceLink from "src/component/RunReferenceLink";
+import { SuccessStateChip } from 'src/component/RunStateChips';
+import Section from 'src/component/Section';
+import TagsList from 'src/component/TagsList';
+import theme from 'src/theme/new';
+
+const StyledSection = styled(Section)`
+    height: fit-content;
+    min-height: 200px;
+    position: relative;
+
+    &:after {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        height: 1px;
+        width: calc(100% + ${theme.spacing(10)});
+        margin-left: -${theme.spacing(5)};
+        background: ${theme.palette.p3border.main};
+    }
+    
+`;
+
+const StyledVertButton = styled(MoreVertButton)`
+    position: absolute;
+    right: 0;
+    top: 12px;
+    transform: translate(50%,0);
+`;
+
+const BoxContainer = styled(Box)`
+    display: flex;
+    flex-direction: row;
+    height: 35px;
+    align-items: center;
+    margin-bottom: ${theme.spacing(3)};
+
+    & .Info {
+        margin-left: ${theme.spacing(2)};
+
+        > div {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            column-gap: ${theme.spacing(2)};
+        }
+    }
+`;
+
+const RunStateContainer = styled(Box)`
+    flex-grow: 0;
+    width: 25px;
+`
+
+const ImportPathContainer = styled(Box)`
+    margin-bottom: ${theme.spacing(3)};
+`
+
+const StyledRunReferenceLink = styled(RunReferenceLink)`
+    font-size: ${theme.typography.fontSize}px ;
+`;
+
+const FunctionName = PipelineTitle
+
+const FunctionSection = () => {
+    return <StyledSection>
+        <Headline>Function Run</Headline>
+        <BoxContainer>
+            <RunStateContainer>
+                <SuccessStateChip size={"large"} />
+            </RunStateContainer>
+            <div className='Info'>
+                <FunctionName>
+                    evaluate_model
+                </FunctionName>
+                <div>
+                    <StyledRunReferenceLink runId={"qwc2ldf"} />
+                    {`Completed in ${Duration("2012-04-23T18:14:23.511Z", "2012-04-23T18:25:43.511Z")} on ${DateTimeLong(new Date())}`}
+                </div>
+            </div>
+        </BoxContainer>
+        <ImportPathContainer>
+            <ImportPath>examples.mnist.train_eval.evaluate_model</ImportPath>
+        </ImportPathContainer>
+        <StyledVertButton />
+        <TagsList tags={['example', 'torch', 'mnist']} />
+        <Docstring docstring={"# Function header line \n                 Some of the function description goes here."} />
```

Comment:
Please initially show more of the docstring, as the partially-obscured docstring title is kinda weird:
![image](https://user-images.githubusercontent.com/1894533/233219115-caabc55d-3b89-416e-b77a-f34c950e39bc.png)

Reply:
The point is that the top panel has a 200px height to match other panels on the left column.
I think I would push for moving the docstring to a dedicated docstring tab.

Reply:
Explaination of the behavior: The blurry thing is because the docstring is taller than the limited ~50px height. So the blurriness hints to the user that there is more below and encourages the user to click the chevron to expand and see more. 

The blurriness will not show if the docstring is short (usually it has to be a single line)

`The point is that the top panel has a 200px height to match other panels on the left column.` Yes indeed. That puts the docstring section limited to ~50px height and no more room to increase.

I will try to put it into a separate tab, then we can compare the two experiences and make a conclusion.

Reply:
Docstring section is now removed. More relaxed spacing in the header section is added.